charts/actions-runner-controller/templates/githubwebhook.serviceMonitor.yaml

```
@@ -8,6 +8,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   name: {{ include "actions-runner-controller-github-webhook-server.serviceMonitorName" . }}
+  namespace: {{ .Values.serviceMonitorNamespace }}
```

It looks like you missed `.metrics` in-between?

```suggestion
  namespace: {{ .Values.metrics.serviceMonitorNamespace }}
```

---

Fixed:
```
helm template . | grep -a100 namespace | grep -b1 test-service
17755-  name: release-name-actions-runner-controller-service-monitor
17818:  namespace: test-service-monitor
17852-spec:
--
18586-  name: release-name-actions-runner-controller-github-w-service-monitor
18658:  namespace: test-service-monitor
18692-spec: